template/src/service.types.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Typescript definitions of actions supported by the {{serviceName}} service
+ *
+ * Copyright Byte Technology 2019. All rights reserved.
+ */
+
+import {
+  GenericEventWithoutPayload as EventNoData,
+  GenericEventWithPayload as Event
+} from 'moleculer-service-ts';
+import { Actions as {{capitalizedServiceName}}Actions } from './api';
+import { ExampleEvent } from './api/events/example.event';
+
+export type ServiceName = '{{serviceName}}';
+
+// Add other services types here, eg;
+// export type ServiceAction = DiscoutActions & UserActions & IotActions;
```

Comment:
Should be be ```&``` or ```|```? It should be union, no?

Reply:
Good point - it should be union `|`